### ISSUE TITLE
Path of  config and rom directory on Linux (OpenSUSE)

### ISSUE THREAD

**Issue by petran79** [86Box/86Box#4963]
### What happened?

I downloaded the Appimage file as the Flatpak version had some issues. 

Problem is I have no clue where 86box stores the config and rom files. 

Even the Flatpak version does not have the roms in the var flatpak application folder 
 ./local/share/flatpak/app/net._86box._86Box/x86_64/stable/44bce6567b09a2101828a3298208da06ac08b9e3e476c89a5c5e1e5e10cd9b64/files/share/86Box/roms is empty

Regarding the Appimage file, I created a  86box/roms folder both in ./home and ./local/share, yet it does not recognize it and gives me a message that the rom folder was not found. 
I do not even where the config folder is located.

### Configuration file

```ini
I could not locate the config file
```

### Operating system

OpenSUSE TW 15.6

### CPU

intel i7 4790k

### 86Box version

4.2.1

### Build architecture

Linux - x64 (64-bit)

### Build type

- [ ] New recompiler
- [ ] Debug build

### Download source

Official website (Jenkins, GitHub)

**Comment by petran79** [comment]
solved. It was inside the Appimage folder

**Comment by eddmanx** [comment]
Questions and usage related matters should be posted in Discussions or on the discord server.
Flatpak is non-official and not supported.

**Comment by OBattler** [comment]
Yes, the config should be in the Appimage folder - though that's with 86box without parameters, which is a deprecated usage. I recommend using -P / --vmpath followed by the path to the emulated machine instead, then the configuration file is going to be located there instead.
